src/sql/magic.py

```diff
@@ -180,6 +181,30 @@ def _mutex_autopandas_autopolars(self, change):
                 setattr(self, other, False)
                 print(f"Disabled '{other}' since '{change['name']}' was enabled.")
 
+    def check_random_arguments(self, line="", cell=""):
+        # check only for cell magic
+        if cell != "":
+            tokens = line.split()
```

Comment:
better to use shlex insted of `line.split`: https://docs.python.org/3/library/shlex.html#shlex.split

Reply:
So, we still need to use` shlex.split() `instead of `line.split()`?